openstack_plugin/resources/network/router.py

```diff
@@ -271,7 +274,8 @@ def add_interface_to_router(openstack_resource, **kwargs):
 @with_compat_node
 @with_openstack_resource(
     OpenstackRouter,
-    existing_resource_handler=_handle_disconnect_external_subnet_from_router)
+    existing_resource_handler=_handle_disconnect_external_subnet_from_router,
+    ignore_unexisted_resource=True)
```

Comment:
@0lvin Why should we ignore if the router does not exist ? why should raise error and what if the user is defining `create_if_missing` which is going to create the resource, so using this param will cause issue not to create the resource and just return